lib/node_modules/@stdlib/math/base/assert/is-negative-integer/manifest.json

```
@@ -0,0 +1,41 @@
+{
+	"options": {},
+	"fields": [
+	  {
+		"field": "src",
+		"resolve": true,
+		"relative": true
+	  },
+	  {
+		"field": "include",
+		"resolve": true,
+		"relative": true
+	  },
+	  {
+		"field": "libraries",
+		"resolve": false,
+		"relative": false
+	  },
+	  {
+		"field": "libpath",
+		"resolve": true,
+		"relative": false
+	  }
+	],
+	"confs": [
+	  {
+		"src": [
+		  "./src/main.c"
+		],
+		"include": [
+		  "./include"
+		],
+		"libraries": [],
+		"libpath": [],
+		"dependencies": [
+		  "@stdlib/math/base/special/floor"
+		]
+	  }
+	]
+  }
+  
```

Should have two-space indentation for the `manifest.json`:

```suggestion
{
  "options": {},
  "fields": [
    {
      "field": "src",
      "resolve": true,
      "relative": true
    },
    {
      "field": "include",
      "resolve": true,
      "relative": true
    },
    {
      "field": "libraries",
      "resolve": false,
      "relative": false
    },
    {
      "field": "libpath",
      "resolve": true,
      "relative": false
    }
  ],
  "confs": [
    {
      "src": [
        "./src/main.c"
      ],
      "include": [
        "./include"
      ],
      "libraries": [],
      "libpath": [],
      "dependencies": [
        "@stdlib/math/base/special/floor"
      ]
    }
  ]
}

```

---

> Should have two-space indentation for the `manifest.json`:

@Planeshifter Ok sir next time I will keep in mind.